Demo/SwiftUI/PaymentsViewModel.swift

```diff
@@ -24,15 +30,23 @@ internal final class PaymentsViewModel: ObservableObject, Identifiable, Presente
         dropInAdvancedFlow.presentDropInComponent()
     }
 
+    internal func presentCardComponent() {
+        cardComponent.presentCardComponent()
```

Comment:
This looks confusing, maybe `cardComponentAdvancedFlowExample` even though a bit long :)

Reply:
`cardAdvancedFlow` ?

Reply:
e7c03971d28f7f4fb10a23fe39580f2dacb88ac4